fernet_fields/fields.py

```diff
@@ -100,6 +90,22 @@ def validators(self):
             del self.__dict__['_internal_type']
 
 
+def get_prep_lookup(self):
+    """Handles lookups for EncryptedField fields"""
+    if self.lookup_name == 'isnull':
+        return super(self.__class__, self).get_prep_lookup()
+    else:
+        raise FieldError("Encrypted fields do not support lookups.")
```

Comment:
@brad Shoot, it is helpful to have the class name and field name in the error message. I'm wondering if there is a way we can keep that?